tests/actions/test_action_typedicts.py

```
@@ -0,0 +1,234 @@
+        )
+
+
+def test_typeddict_fields_match_action_fields() -> None:
+    """Test that TypedDict fields match the corresponding action class fields."""
+    # Fields that are excluded from TypedDicts (internal/metadata fields)
+    excluded_fields = {"category", "description", "param", "press_enter", "text_label"}
+
+    errors: list[str] = []
+    mapping = _get_action_to_typeddict_mapping()
+
+    for action_type, (action_cls, typeddict_cls) in mapping.items():
```

Ruff (0.14.8)

Loop control variable `action_type` not used within loop body

Rename unused `action_type` to `_action_type`

(B007)